src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Address.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources xmlns="https://api-platform.com/schema/metadata"
```

Comment:
Do we need this configuration? Serialization shouldn't be enough? if I see correctly we don't use this `get` method anywhere.